py_gtfs_rt_ingestion/ingest.py

```diff
@@ -3,21 +3,21 @@
 import argparse
 import json
 import os
-import pyarrow.parquet as pq
+import pyarrow as pa
 import sys
-import tempfile
 
-from pathlib import Path
+from multiprocessing import Pool
```

Comment:
multiprocessing is back in here.

we're not using tempfiles anymore. reading directly from s3.

not using Path either, keeping things as strings to make it more uniform between s3 and local filepaths.